_posts/2019-12-20-differential-privacy/differential_privacy.Rmd

```diff
@@ -0,0 +1,470 @@
+ 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019diffpriv
+bibliography: bibliography.bib
+date: 12-20-2019
+categories:
+  - Security
+  - Keras
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/cat.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+
+What could be treacherous about summary statistics? 
+
+The famous _cat overweight_ study (X. et al., 2019) showed that as of May 1st, 2019, 32 of 101 domestic cats held in Y., a cozy Bavarian village, were overweight. Even though I'd be curious to determine if my aunt G.'s cat (a happy resident of that village) has been fed too many treats and has accumulated some excess pounds, from the results reported in that study I can't know.
+
+Then, six months later, out comes a new study, ambitious to earn scientific fame. The authors report that of 100 cats living in Y., 50 are striped, 31 are black, and the rest are white; the 31 black ones are all overweight. Now, I happen to know that with one exception, no new cat joined the community, and no cat left. _But_, my aunt moved away to town, to a retirement home, chosen of course for the possibility to bring one's cat.
+
+What have I just learned? My aunt's cat is overweight. (Or was, at least, before they moved to the retirement home.)
+
+Even though none of the studies reported anything but summary statistics, I was able to infer individual-level facts, by connecting both studies and adding in another piece of information I had access to. 
+
+In reality, mechanisms like the above -- technically called _linkage_ -- have been shown to lead to privacy breaches many times, thus defeating the purpose of _database anonymization_ seen as a panacea in many organizations. A more promising alternative is offered by the concept of _differential privacy_.
+
+## Differential Privacy
+
+In differential privacy (DP)^[We'll be using DP as an acronym for both the noun phrase "differential privacy" and the adjective phrase "differentially private".][@Dwork2006], privacy is not a property of what's in the database; it's a property of how query results are delivered. 
+
+Intuitively paraphrasing results from a domain where results are communicated as theorems and proofs [@dwork2006differential][@Dwork], the only achievable (in a lossy but quantifiable way) objective is that from queries to a database, nothing more should be learned about an individual in that database than if they hadn't been in there at all.[@primer] 
+
+What this statement does is caution against overly high expectations: Even if query results are reported in a DP way (we'll see how that goes in a second), they enable some probabilistic inferences about individuals in the respective population. (Otherwise, why conduct studies at all.)
```

Comment:
```suggestion
What this statement does is caution against overly high expectations: Even if query results are reported in a DP way (we'll see how that goes in a second), they enable some probabilistic inferences about individuals in the respective population. (Otherwise, why conduct studies at all?)
```